dev-lang/python/python3.9-3.9.18.recipe

```diff
@@ -53,7 +53,7 @@ REQUIRES="
 	lib:liblzma$secondaryArchSuffix
 	lib:libncurses$secondaryArchSuffix
 	lib:libsqlite3$secondaryArchSuffix
-	lib:libssl$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix >= 3
```

Comment:
this is normally not needed.

Reply:
same for other recipes.

Reply:
Ok, fixed by removing the constraint on lib: and keeping it only on devel: